test/unit/spec/appWidgets/commonSpec.js

```diff
@@ -5,50 +5,48 @@ define(['common/events', 'common/ui', 'kb_common/html', 'widgets/appWidgets2/com
     WidgetCommon
 ) => {
     'use strict';
-    const div = html.tag('div');
-    const button = html.tag('button');
-    const container = document.createElement('div');
-    const events = Events.make({ node: container });
-    const ui = UI.make({ node: container });
+    const div = html.tag('div'),
```

Comment:
refactored this file to be in line with the paradigms we have used on the Truss project